src/rfbzero/degradation.py

```diff
@@ -6,71 +6,111 @@
         raise NotImplementedError
 
 
-class ChemicalDegradation(DegradationMechanism):
+class ChemicalDegradationOxidized(DegradationMechanism):
     """
-    Provides an N-th order chemical degradation mechanism.
+    Provides an N-th order chemical degradation mechanism for an oxidized species.
 
     Parameters
     ----------
     rate_order : int
         Rate order for chemical degradation reaction.
     rate_constant : float
         N-th order rate constant of chemical oxidation (unit is rate order-dependent).
-    species : str
-        Species ('red' or 'ox') undergoing chemical degradation.
 
     """
 
-    def __init__(self, rate_order: int, rate_constant: float, species: str = 'red') -> None:
+    def __init__(self, rate_order: int, rate_constant: float) -> None:
         self.rate_order = rate_order
         self.rate_constant = rate_constant
-        self.species = species
 
-        if not isinstance(self.rate_order, int) or self.rate_order < 0:
-            raise ValueError("'rate_order' must be an integer")
+        if self.rate_order < 0:
+            raise ValueError("'rate_order' must be >= 0")
 
         if self.rate_constant <= 0.0:
             raise ValueError("'rate_constant' must be > 0.0")
 
-        if self.species not in ['red', 'ox']:
-            raise ValueError(f"Invalid value: {self.species}. 'species' options: 'red', 'ox' ")
-
-    def degrade(self, c_ox: float, c_red: float, timestep: float) -> tuple[float, float]:
+    def degrade(self, c_ox: float, c_red: float, time_step: float) -> tuple[float, float]:
         """
-        N-th order chemical degradation of N*[redox-active species] --> [redox-inactive species].
-        Returns updated concentrations. Concentration may be unchanged if species does not degrade.
+        Applies N-th order chemical degradation of N*[redox-active species] --> [redox-inactive species] at each
+        time step. Returns updated concentrations; concentration may be unchanged if species does not degrade.
 
         Parameters
         ----------
         c_ox : float
             Concentration of oxidized species (M).
         c_red : float
             Concentration of reduced species (M).
-        timestep : float
+        time_step : float
             Time interval size (s).
 
         Returns
         -------
+        c_ox : float
+            Updated concentration of oxidized species (M).
+        c_red : float
+            Unchanged concentration of reduced species (M).
+
+        """
+
+        c_ox -= (time_step * self.rate_constant * (c_ox ** self.rate_order))
+        return c_ox, c_red
+
+
+class ChemicalDegradationReduced(DegradationMechanism):
+    """
+    Provides an N-th order chemical degradation mechanism for a reduced species.
+
+    Parameters
+    ----------
+    rate_order : int
+        Rate order for chemical degradation reaction.
+    rate_constant : float
+        N-th order rate constant of chemical oxidation (unit is rate order-dependent).
```

Comment:
similar to above, now say `...constant of chemical degradation of reduced species`